org.gnomeradio.Radio.appdata.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<application>
+  <id type="desktop">org.gnome.Radio.desktop</id>
+  <name>GNOME Radio 64 for GNOME 44</name>
+  <summary>Listen to National Public Radio on GNOME 44</summary>
+  <licence>CC0</licence>
```

Comment:
This is an invalid tag. Also the license identifier is invalid too.

Reply:
Which tag is invalid?